### ISSUE TITLE
Export to CSV?

### ISSUE THREAD

**Issue by vamshi2017** [EBGreen684/Compare-ADGroupMembership#1]
hi,
Should removing the username part from the script just give the output of machine names? and adding | Export-csv at the end gives the csv file?

thanks,
vamshi.

**Comment by EBGreen684** [comment]
Well, user objects are not the same thing as computer objects. Right now it
is written to work with user objects. Do you want it to be computer objects?

**Comment by vamshi2017** [comment]
yes, i want it to be computer objects. and also is there a better way to compare the first reference groups also to be multiple.

Like, comparing Group1 or group2 members with group3 or group 4 or so on.

**Comment by EBGreen684** [comment]
I can make these changes tonight.
